docs/contributing/core-services.md

```diff
@@ -1,6 +1,6 @@
 # Terraform AWS Provider Core Services
 
-Core Services are AWS services we have identified as critical for a large majority of our users. Our goal is to continually increase the depth of coverage for these services. We will work to prioritize features and enhancements to these services in each weekly release, even if they are not necessarily highlighted in our quarterly road map.
+Core Services are AWS services we have identified as critical for a large majority of our users. Our goal is to continually increase the depth of coverage for these services. We will work to prioritize features and enhancements to these services in each weekly release, even if they are not necessarily highlighted in our quarterly roadmap.
```

Comment:
Can we define what those services are explicitly?

Reply:
Great suggestion! I'll push a quick edit to link out to [core services](https://github.com/hashicorp/terraform-provider-aws/blob/main/docs/contributing/core-services.md)

Reply:
Oh, I think I may have misunderstood here. Where were you suggesting to add the explicit list of core services -- to the prioritization guide?

Reply:
🎉 Triple reply! I think I follow you now. I pushed a change to the `core-services.md` file as a part of this PR just to make our use of `roadmap` vs `road map` consistent. There's an explicit list of the core services below this edited line in the same document, which we do link out to from the prioritization guide already.